htdocs/projet/tasks/task.php

```diff
@@ -484,55 +484,60 @@
 			dol_fiche_end();
 		}
 
-
-		if ($action != 'edit')
+		$parameters = array();
```

Comment:
Sure, that's why it must be into the div.
All buttons are into the <div tabsAction
All
print '<a class="butAction"
and
Hooks to add button should be inside the
<div tabsAction
This div means "begin part with buttons".